CommonUI/Sources/FeatureView/MovieDetails/MovieDetailsView.swift

```diff
@@ -76,5 +80,15 @@ public struct MovieDetailsView: View {
         .onAppear {
             viewModel.fetchFavorite()
         }
+        .onAppear {
```

Comment:
this business logic is too complicated for a view. It should be moved to the ViewModel. Otherwise it cannot be properly tested